rllib/connectors/connector_pipeline_v2.py

```diff
@@ -77,16 +73,14 @@ def __call__(
         # Loop through connector pieces and call each one with the output of the
         # previous one. Thereby, time each connector piece's call.
         for connector in self.connectors:
-            timer = self.timers[str(connector)]
-            with timer:
-                data = connector(
-                    rl_module=rl_module,
-                    data=data,
-                    episodes=episodes,
-                    explore=explore,
-                    shared_data=shared_data,
-                    **kwargs,
-                )
+            data = connector(
```

Comment:
Simplified, we have NOT yet used these timer stats at all in our results dicts, so thus far these were a waste of time.